README.md

```diff
@@ -76,35 +111,104 @@ openssl rand -base64 32
 
 Save this value for the next step.
 
-The terraform module will create a Secret Manager secret in the project provided with the name `github-webhook-secret`. Navigate to the Google Cloud dashboard for Secret Manager and add a new revision with this generated value. Note there will be another secret (`github-private-key`) in Secret Manager that will be addressed in the next section.
+The terraform module will create a Secret Manager secret in the project provided with the name `-`. Navigate to the Google Cloud dashboard for Secret Manager and add a new revision with this generated value. Note there will be another secret (`github-private-key`) in Secret Manager that will be addressed in the next section.
 
-## Create github private key secret
+### Create github private key secret
 
 The terraform module will create a Secret Manager secret in the project provided with the name `github-private-key`. Convert your downloaded key from when you created your GitHub App and convert it into a string using the following comman in your terminal. If you didn't create a key earlier when creating your GitHub App or lost your key, go to your GitHub App and create a new one and delete older keys.
 
+In the GitHub App settings, under the Private Keys section,
+1. Click Generate a private key. This will add a `.pem` file to your Downloads.
+The contents of the .pem file is the private_key, including the BEGIN and END.
+It should look something like,
+
+    ```
+    -----BEGIN RSA PRIVATE KEY-----
+    SOME-SUPER-SECRET-
+    SHHHHHHHHHHHHHH-
+    KEEP-THIS-A-SECRET
+    -----END RSA PRIVATE KEY-----
+    ```
+
+Copy the contents of the file:
 ```shell
 cat location/to/private/key.private-key.pem | pbcopy
 ```
 
-Navigate to the Google Cloud dashboard for Secret Manager and add a new revision with this generated value to `github-private-key`. 
+### Upload the secrets
+Navigate to the Google Cloud dashboard for Secret Manager and add a new revision
+with the generated values to their corresponding secret ID's.
+- `github-webhook-secret`
```

Comment:
Didn't we add this in the previous step?

Reply:
Removed the "previous step" documentation, in favor of uploading the secrets together at the same time at this step.